### ISSUE TITLE
Single column line length is too long 

### ISSUE THREAD

**Issue by kristinashu** [mozilla/mofo-bootstrap#106]
Main usability problem here is that there are too many words per line, which makes it difficult to read. I think we need to do two things:
1) Increasing the body text size _(slightly)_
2) Decrease the max content width for single column layouts

Some examples:

This is how it looks on the [demo page](https://mozilla.github.io/mofo-bootstrap/demo/): 

<img width="1412" alt="screen shot 2017-02-17 at 1 17 50 pm" src="https://cloud.githubusercontent.com/assets/1068783/23083637/882a0962-f513-11e6-83f3-b7ef2f674227.png">

and the [NSF site](https://mozilla.github.io/nsf-challenge/) ([related ticket](https://github.com/MozillaFoundation/NSF-wireless-innovation-challenge/issues/28)): 

<img width="1414" alt="screen shot 2017-02-17 at 1 06 55 pm" src="https://cloud.githubusercontent.com/assets/1068783/23083343/15cee0c8-f512-11e6-8a5a-480b30077681.png">

Width is better on some pages of the [science site](https://science.mozilla.org/programs/fellowships) but too wide on others, should be consistent ([related ticket](https://github.com/mozilla/science.mozilla.org/issues/340)):

<img width="1414" alt="screen shot 2017-02-17 at 1 25 36 pm" src="https://cloud.githubusercontent.com/assets/1068783/23083865/9fa04d6c-f514-11e6-9a44-622eba2f61c9.png">

<img width="1415" alt="screen shot 2017-02-17 at 1 07 18 pm" src="https://cloud.githubusercontent.com/assets/1068783/23083367/29468188-f512-11e6-88b8-43126b0a4e6a.png">

Looks a little better on the Network site because width is slightly narrower and this new font might be slightly bigger looking than fira..

<img width="1414" alt="screen shot 2017-02-17 at 1 08 37 pm" src="https://cloud.githubusercontent.com/assets/1068783/23083427/6c7c615c-f512-11e6-81b7-b25feb09e610.png">

It does look like the boostrap site uses some narrower widths, which look good: 

https://blog.getbootstrap.com/

<img width="1415" alt="screen shot 2017-02-17 at 1 59 54 pm" src="https://cloud.githubusercontent.com/assets/1068783/23084880/7c5b1418-f519-11e6-805d-18a7eb3f3af1.png">

http://v4-alpha.getbootstrap.com/

<img width="1416" alt="screen shot 2017-02-17 at 2 01 06 pm" src="https://cloud.githubusercontent.com/assets/1068783/23084896/93866264-f519-11e6-8d4b-e43ce1985a81.png">

@taisdesouzalessa @sabrinang @alanmoo any thoughts what sizes/widths would be idea and also what is possible to do engineering wise, without having to do to many overrides on Bootstrap?

**Comment by taisdesouzalessa** [comment]
Good point @kristinashu. I agree that line width can and should be smaller. Usually it is recommended 75 characters. 

For the Network website, I would recommend to use [the mockup](https://invis.io/ZJ9YTAGEA#/216715351_About) as a reference
<img width="1003" alt="screen shot 2017-02-17 at 2 21 58 pm" src="https://cloud.githubusercontent.com/assets/22157921/23085511/09d19d50-f51d-11e6-9144-c5419ffdd04f.png">

We have an average of 85 characters per line in the Network site. We achieved that by using only 8 columns of the grid and also skipping the first column. We also added an illustration on the right side to make sure the layout is balanced (we just have a placeholder for now but it will become an illustration).

I like the idea of narrower widths for single columns. Easy to read, as you mentioned and it follows the best practices.

**Comment by alanmoo** [comment]
With regards to the width of the lines, this isn't a Bootstrap issue at all; Designs should simply describe how many columns (out of a possible 12) a block of text should take up at various screen sizes.

For example: 12 columns wide up to medium viewports, then 10 columns for large and XL.

We do have the ability to change the viewport breakpoints and the width of the grid at various breakpoints, though I don't think that's necessary here.

For reference, [these are the defaults](https://v4-alpha.getbootstrap.com/layout/grid/#variables).

As for the font size, would we want to change the base font size at various screen sizes for every site, or should that be on a per-site basis? There might be some work to be done to enable that, but for the most part it would be something that must be kept in mind when building a site out (i.e. always using em/rem units, not specific pixel sizes for fonts/margins/padding.)

**Comment by kristinashu** [comment]
All sounds excellent! 

Ideally, base font size would be the same on all our sites.
